packages/worker/src/worker-options.ts

```diff
@@ -686,6 +705,7 @@ export function addDefaultWorkerOptions(options: WorkerOptions): WorkerOptionsWi
     reuseV8Context: reuseV8Context ?? false,
     debugMode: debugMode ?? false,
     interceptors: appendDefaultInterceptors({}),
+    nonStickyToStickyPollRatio: nonStickyToStickyPollRatio ?? 0.2,
```

Comment:
nit: Should probably add some boundary checks here, either here or in core. AFAICS, someone could set `nonStickyToStickyPollRatio: 20.0` and core would just happily start with 20 times the expected number of pollers...

Reply:
This should be validated on the core side IMHO, less logic for every lang SDK.